tests/Functional/RetrieveAutocompleteItemsActionTest.php

```diff
@@ -19,9 +19,11 @@ final class RetrieveAutocompleteItemsActionTest extends BaseFunctionalTestCase
 {
     public function testAutocomplete(): void
     {
-        $this->client->request(Request::METHOD_GET, '/admin/core/get-autocomplete-items?q=autocompletion&_per_page=10&_page=1&uniqid=s608eac968661e&admin_code=Sonata%5CDoctrineORMAdminBundle%5CTests%5CApp%5CAdmin%5CBookWithAuthorAutocompleteAdmin&field=author');
+        $this->client->request(Request::METHOD_GET, '/admin/core/get-autocomplete-items?q=autocompletion&_per_page=10&_page=1&uniqid=s608eac968661e&_sonata_admin=Sonata%5CDoctrineORMAdminBundle%5CTests%5CApp%5CAdmin%5CBookWithAuthorAutocompleteAdmin&field=author');
```

Comment:
I had to add conflict with `sonata-project/admin-bundle` `alpha-2`, otherwise this test fails because in `alpha-2`, `get-autocomplete-items` route expects a `admin_code` and in the `4.0@dev` version expects `_sonata_admin`.

Other solution could be to add both parameters and leave a `NEXT_MAJOR` message without adding that conflict.

Reply:
The conflict seems ok